src/modules/database/generators/DmlGenerator.cpp

```diff
@@ -0,0 +1,103 @@
+#include "DmlGenerator.h"
+#include "ColumnTypeValueProvider.h"
+
+#include "faker-cxx/Number.h"
+
+namespace faker
+{
+
+std::string DmlGenerator::generate()
+{
+        // get a random table name
+    std::string table = getTable();
+
+    // get random columns
```

Comment:
consider removing those comments as they are not bringing any information other than code